site/content/guides/claude-code-monitoring.md

```diff
@@ -0,0 +1,415 @@
+npm create checkly@latest
+```
+
+Next, install [Claude Code](https://www.anthropic.com/claude-code) globally with 
+
+```bash
+npm install -g @anthropic-ai/claude-code
+```
+
+and from the root folder of your project,  start Claude Code with the `claude` command.
+
+You'll be prompted to sign in to or sign up for an Anthropic account.
+
+We recommend using [the Visual Studio Code plugin for Claude](https://marketplace.visualstudio.com/items?itemName=anthropic.claude-code), which will make it easier to review changes to code files with its side-by-side viewer (the plugin does a lot more besides that we won't cover in this guide).
+
+### Add the Playwright MCP
+
+You can give Claude Code access to a browser by adding the [Playwright MCP](https://github.com/microsoft/playwright-mcp) to your local configuration with 
+
+```bash
+claude mcp add playwright npx @playwright/mcp@latest 
+```
+This will let us ask Claude Code to do things like crawling a particular website and acting on what it finds by opening the site in a browser.
+
+## Give Claude Code Some Context
+Everything we write into the CLI interface is part of a 'prompt,' and by default, Claude Code will add a bit of information to what we submit as 'context' on that prompt. Ideally, our requests written into Claude Code would include context about our specific project: its structure, language, and even its purpose. Because of the `npm create` command we ran above, our folder already has a boilerplate version of a Checkly project. We can't feed in all of these files as context, but we can add a summary of our whole project as context by opening Claude Code and running the command:
+
+```{title="Prompt:"}
+/init
+```
+
+Claude Code will scan the whole project, write a summary, and then store that summary in a local CLAUDE.md file. Claude Code will read that file and add it to our prompt every time it's run within this project from now on.
+
+Let's add a bit more to this context, we want test files that make sense as a [Checkly construct](https://www.checklyhq.com/docs/cli/constructs-reference/). Let's add [Checkly's AI rules file](https://www.checklyhq.com/docs/ai//use-checkly-with-ai-ide/#claude-code) to our CLAUDE.md context file:
+
+```bash
+mkdir -p .claude &&
+curl -o .claude/checkly.rules.md https://www.checklyhq.com/docs/ai/checkly.rules.md -L 
+echo "- examine checkly.rules.md for code generation rules" >> .claude/CLAUDE.md
+```
+
+After this you'll need to exit and restart Claude Code for it to pick up changes to our context file.
+
+## Create new URL Monitors
+
+We'll start with something easy. A URL monitor that will check an HTTP endpoint and alert if it doesn't receive the right status code. With the context set from the section above, try a command like:
+
+```{title="Prompt:"}
+Create a new URL monitor for the site https://danube-webshop.herokuapp.com/ that runs every five minutes and doesn't follow redirects.
+```
+
+After a few seconds, Claude Code will produce this:
+
+```ts
```

Comment:
This is the only file code block with a file name header. Is this in purpose?

Reply:
Just trying to keep the visual clear, it's the only code block where we don't refer back to the title.